client/dashboard/data/domain-nameservers.ts

```diff
@@ -0,0 +1,27 @@
+import wpcom from 'calypso/lib/wp';
+
+export async function fetchDomainNameservers( domainName: string ): Promise< string[] > {
```

Comment:
[nit] It's nice to be consistent -- is it `name-servers` or `nameservers`? We use both variants in this PR (filenames and variable/function names)